src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -75,7 +75,7 @@
         public static final int FLYWHEEL_1 = 0, FLYWHEEL_2 = 0, TURRET = 0, HOOD = 0;
 
         // intake can ids are range 30-39
-        public static final int INTAKE_1 = 0, INTAKE_2 = 0, INTAKE_UP = 0, INTAKE_DOWN = 0;
+        public static final int INTAKE_1 = 32, INTAKE_2 = 34, INTAKE_UP = 36, INTAKE_DOWN = 38;
```

Comment:
Intake_up and down are solenoid ports, which have to be somewhere in the range of the pneumatic controller we're using (usually something like 0-15) and non-overlapping with the other subsystems.  I'd also recommend updating the constant names differential motor IDs from pneumatic IDs.